src/coreclr/vm/gcinfodecoder.cpp

```diff
@@ -125,7 +125,6 @@ GcInfoDecoder::GcInfoDecoder(
     }
 
     m_IsVarArg                 = headerFlags & GC_INFO_IS_VARARG;
-    int hasSecurityObject      = headerFlags & GC_INFO_HAS_SECURITY_OBJECT;
```

Comment:
Can this flag be removed from the enum? And should crossgen2 GcInfo.cs get a similar change?

Reply:
> And should crossgen2 GcInfo.cs get a similar change?

GcInfo.cs is part of ILCompiler.Reflection.ReadyToRun. ILCompiler.Reflection.ReadyToRun should be able to inspect old R2R images, so it should stay there. For example, it has `MIN_GCINFO_VERSION_WITH_REV_PINVOKE_FRAME` that has been deleted from the runtime.